src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.external {
+  box-sizing: border-box;
+  width: 350px;
+  border: 1px solid #e75551;
+  background-color: #308dc0;
+  padding: 10px;
+  margin: 20px 20px 40px;
+}
+
+.middle {
+  border: 1px solid #e75551;
+  background-color: #ffa500;
+  padding: 10px;
+}
+
+p {
+  border: 1px solid #e75551;
+  background-color: #ddd;
+  padding: 10px;
+  margin: 0;
+}
```

Comment:
if you see repeated styles you have to think about how to optimize them so that there is no such issue